### PR TITLE
Added the ability to change the name of the main branch

### PR DESCRIPTION
**Pull request by vadim8960**
+ Edited esbuild.config.mjs because on my local machine don't build

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -11,24 +11,26 @@ interface GHSyncSettings {
...
 	isSyncOnLoad: boolean;
+	mainBranchName: string;
 }
 
 const DEFAULT_SETTINGS: GHSyncSettings = {
 	remoteURL: '',
 	gitLocation: '',
 	syncinterval: 0,
 	isSyncOnLoad: false,
+	mainBranchName: 'master'
```
Comment (serg-kharin) [inline main.ts:22]: I guess it has to be 'main' coz it was a previous default

Comment (vadim8960) [reply main.ts:22]: You're right, I forgot to correct it

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
 		"typescript": "4.7.4"
 	},
 	"dependencies": {
+		"events": "^3.3.0",
```
Comment (serg-kharin) [inline package.json:25]: Why do we need this dependency?

Comment (vadim8960) [reply package.json:25]: These were attempts to run plugin. For some reason, I couldn't run it without this library)

Comment (serg-kharin) [reply package.json:25]: I think it is not good practice to commit PC-specific code to public repository....

Comment (vadim8960) [reply package.json:25]: ok, I'll delete it in a couple of hours

**Pull request closed without merging**